packages/web-frontend/src/utils/formatDateForApi.js

```diff
@@ -5,13 +5,12 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import moment from 'moment-timezone';
+import moment from 'moment';
 
 const DATE_FORMAT = 'YYYY-MM-DD';
 
-export const formatDateForApi = (date, timezone) => {
```

Comment:
The timezone prop was never being used and removing support for it means that we can completely remove moment-timezone as a dependency from web-frontend.

Reply:
Nice one. There is actually a function in `utils` called `formatDateForApi` which looks like it does the same thing. Might not be worth swapping it out tho, since we are going to eventually get rid of `web-frontend`

Reply:
Good spotting but yeah probably not worth it at this point in web-frontend.